src/Conference/ConferenceDetails.test.js

```diff
@@ -27,8 +26,50 @@ describe("ConferenceDetails", () => {
     topic: "Marketing"
   };
 
-  it("renders conference details", async () => {
-    expect.assertions(6);
+  const mockCookie = {
+    sessionToken: {
+      userId: 1,
+      token: "3ecb9d1d-863f-4207-b076-d868e6544c3b"
+    }
+  };
+
+  it("renders a logged-in conference details page", async () => {
+    expect.assertions(8);
+
+    const history = createMemoryHistory();
+    history.push("/1");
+
+    let wrapper;
+    act(() => {
+      wrapper = mount(
+        <Router history={history}>
+          <ConferenceDetails
+            conference={mockData}
+            id="1"
+            allCookies={mockCookie}
+          />
+        </Router>
+      );
+    });
+    wrapper.update();
+    expect(wrapper.find(".name").get(0).props.children).toBe(mockData.name);
```

Comment:
at this point you should make a shorthand for this.
`const  findElement = (identifier: string) => wrapper.find(identifier).get(0).props.children)`
then
`expect(findElement('name')).toBe(mockData.name);`
If you then use this in multiple places you can make it a util and import in your tests

(I've not tested this and just wrote it here, may not be perfect 😅)